bindings/matlab/test_camera_ethernet.m

```diff
@@ -0,0 +1,10 @@
+
+%% The second device will always be a generic ethernet device where the ip
```

Comment:
.. where the ip specified .. The ip "is" specified. Seems that you're missing a verb or something

Reply:
Fixed. Thanks for noticing!